acceptance-tests/tests/src/test/resources/jsonrpc/engine/test-cases/10_shanghai_invalid_null_withdrawals_prepare_payload.json

```diff
@@ -0,0 +1,28 @@
+{
+  "request": {
+    "jsonrpc": "2.0",
+    "method": "engine_forkchoiceUpdatedV2",
+    "params": [
+      {
+        "headBlockHash": "0x3559e851470f6e7bbed1db474980683e8c315bfce99b2a6ef47c057c04de7858",
+        "safeBlockHash": "0x3559e851470f6e7bbed1db474980683e8c315bfce99b2a6ef47c057c04de7858",
+        "finalizedBlockHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+      },
+      {
+        "timestamp": "0x10",
+        "prevRandao": "0x0000000000000000000000000000000000000000000000000000000000000000",
+        "suggestedFeeRecipient": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
+      }
+    ],
+    "id": 67
+  },
+  "response": {
+    "jsonrpc" : "2.0",
+    "id" : 67,
+    "error" : {
+      "code" : -32602,
+      "message" : "Invalid params"
```

Comment:
nit: I like this test, and think it would be good to test the equivalent functionality for newPayloadV2 as well.

Reply:
do you mean an invalid test for newPayloadV1 including withdrawals?

Reply:
This is testing fcuV2 with null withdrawals
We could test newPayloadV2 with null withdrawals as well

Reply:
Misread that first time as FCU. Yeah, that makes sense and is easy to do.

Reply:
done